src/utils/provider.ts

```diff
@@ -2,12 +2,13 @@ import defu from 'defu'
 import axios from 'axios'
 import bodyParser from 'body-parser'
 import requrl from 'requrl'
+import qs from 'qs'
 
 export function assignDefaults(strategy, defaults) {
   Object.assign(strategy, defu(strategy, defaults))
 }
 
-export function addAuthorize(nuxt, strategy) {
+export function addAuthorize (nuxt, strategy, useForms = false) {
```

Comment:
Why is `useForms` needed?

Reply:
This is to let axios know when sending the oauth token exchange that it must send as a urlencoded form since Discord only accepts type 'x-www-form-urlencoded' content

> In accordance with the relevant RFCs, the token and token revocation URLs will only accept a content type of x-www-form-urlencoded. JSON content is not permitted and will return an error.

[Discord OAuth2 Documentation](https://discord.com/developers/docs/topics/oauth2)

It is defaulted to false so it doesn't affect the other providers under the hood